modules/computer/display.c

```diff
@@ -28,18 +28,22 @@ get_glx_info(DisplayInfo *di)
     if (g_spawn_command_line_sync("glxinfo", &output, NULL, NULL, NULL)) {
 	gchar **output_lines;
 	gint i = 0;
+	gint cpd = 0;
 
 	for (output_lines = g_strsplit(output, "\n", 0);
 	     output_lines && output_lines[i];
 	     i++) {
-	    if (strstr(output_lines[i], "OpenGL")) {
+	    if (strstr(output_lines[i], "OpenGL")) 
+	    {
+		if ( strstr(output_lines[i], "profile") ) {    cpd = 1;    }
 		gchar **tmp = g_strsplit(output_lines[i], ":", 0);
 
 		tmp[1] = g_strchug(tmp[1]);
 
 		get_str("OpenGL vendor str", di->ogl_vendor);
 		get_str("OpenGL renderer str", di->ogl_renderer);
-		get_str("OpenGL version str", di->ogl_version);
+		if ( corep > 0 ) {    get_str("OpenGL core profile version str", di->ogl_version);    }
```

Comment:
cpd cpd! ok ok its due i was triying in varous machines, you can made modifications on my pull directly.. github have the feature that "able to made modification by others"